lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -170,6 +170,12 @@ def self.included(includer)
               Chef::Config[:knife][:bootstrap_vault_item][vault].push(item)
               Chef::Config[:knife][:bootstrap_vault_item]
             }
+
+          option :tags,
+            :long => "--tags TAGS",
+            :description => "Comma separated list of tags to apply to the node",
+            :proc => lambda { |o| o.split(/[\s,]+/) },
+            :default => []
```

Comment:
I see now, it's for the `first_boot` change. I think this should go in a separate PR, too.